admin/index.html

```diff
@@ -10,6 +13,96 @@
   <body>
     <!-- Include the script that builds the page and powers Netlify CMS -->
     <script src="https://unpkg.com/netlify-cms@^2.0.0/dist/netlify-cms.js"></script>
+    <script>
+      const ProgramUpdatePreview = createClass({
```

Comment:
If this is React, `createClass` was [effectively deprecated in 2017](https://legacy.reactjs.org/blog/2017/04/07/react-v15.5.0.html) and [later removed](https://legacy.reactjs.org/blog/2017/09/26/react-v16.0.html). It might save us some future refactoring if we made these [function components](https://react.dev/learn/your-first-component#defining-a-component) from the start?